control_node:
  ros__parameters:
    lookahead_distance: 1.5
    goal_tolerance: 0.1
    linear_speed: 0.5
    max_steering_angle: 1.5